src/test/java/com/google/api/codegen/protoannotations/testdata/library.yaml

```diff
@@ -1,6 +1,6 @@
 type: google.api.Service
 config_version: 1
-name: library-example.googleapis.com
+name: library-example.googleapis.com:1234
```

Comment:
This reflects the main library.yaml service yaml used by all the other baseline tests.